packages/editor-ui/src/plugins/i18n/locales/en.json

```diff
@@ -470,6 +470,8 @@
 	"nodeErrorView.stack": "Stack",
 	"nodeErrorView.theErrorCauseIsTooLargeToBeDisplayed": "The error cause is too large to be displayed",
 	"nodeErrorView.time": "Time",
+	"nodeErrorView.inputPanel.title": "A previous node errored",
+	"nodeErrorView.inputPanel.text": "View error",
```

Comment:
could we rename these to something more descriptive? maybe something like `nodeErrorView.inputPanel.previousNodeError`